.bazelrc

```diff
@@ -0,0 +1,5 @@
+startup --host_jvm_args=-Dbazel.DigestFunction=sha256
+
+build --remote_local_fallback
+build --remote_http_cache=http://bazel-cache.kubevirt-prow.svc.cluster.local:8080/kubevirt.io/kubevirt
```

Comment:
I think that you need to inject it during the test-runs. If you put it here by default, then when I locally use bazel, it would try to reatch that cache. I mean it would still contiune since it is not reachable from outside, but still.